src/acl/Acl.h

```diff
@@ -82,7 +82,11 @@ class ACL
 
     virtual void prepareForUse() {}
 
-    SBufList dumpOptions(); ///< \returns approximate options configuration
+    // TODO: Find a way to make options() and this method constant
+    /// Prints aggregated "acl" (or similar) directive configuration, including
+    /// the given directive name, ACL name, ACL type, and ACL parameters. The
+    /// printed parameters are collected from all same-name "acl" directives.
+    void dumpWhole(const char *directiveName, std::ostream &);
```

Comment:
We should not name this method dump() because there is a virtual dump() method already. In a sense, maybe that inconsistency is a good thing because this method has to print (an equivalent of) more than one "acl" directive (due to acl parameter aggregation by ACL objects).

Reply:
IMO that is fine for now. The whole "dump" thing is going away with the manager report project.